scalameta/parsers/js/src/main/scala/scala/meta/parsers/JSFacade.scala

```diff
@@ -85,7 +85,7 @@ object JSFacade {
           case Some(dialect) => Right(dialect)
           case None => Left(s"'$dialectStr' is not a valid dialect.")
         }
-      case None => Right(dialects.Scala211)
+      case None => Right(dialects.Scala213)
```

Comment:
ast explorer was already setting Scala213 as the default parser, but this is just in case someone uses it standalone